org.moreunit.plugin/src/org/moreunit/wizards/NewTestCaseWizard.java

```diff
@@ -70,17 +70,8 @@ public void createPageControls(Composite pageContainer)
 
     private void configurePageOne()
     {
-        // Eclipse 3.1.x does not support junit 4
```

Comment:
The reason I have deleted this code, because another similar call was already done in the init method here:
https://github.com/MoreUnit/MoreUnit-Eclipse/blob/b6c763fe2b2026ecefb62d1f5e903eaf3b71a1b2/org.moreunit.plugin/src/org/moreunit/wizards/MoreUnitWizardPageOne.java#L257

The only difference could be the first parameter.
 - In case of init, it's true if the junit Test annotation is on the classpath
 - In this case, it's true if the user set Junit4 in the preference

In any case, the only difference the first parameter makes are these calls:
`setSuperClass("java.lang.Object", false);` and `unit4Toggle.setSelection(isJUnit4);`

This should not cause NoSuchMethodError, any other place would stop the plugin from initializing the dialog anyways.

As I see the button is always enabled, because there is no call in the codebase with `false` as the second parameter, and the preference value is always the selected, even if there was a NoSuchMethodError exception here, because the radio button selection is done here:
https://github.com/MoreUnit/MoreUnit-Eclipse/blob/b6c763fe2b2026ecefb62d1f5e903eaf3b71a1b2/org.moreunit.plugin/src/org/moreunit/wizards/MoreUnitWizardPageOne.java#L559

So based on these, I have assumed, this is some dead branch now, but there was quite a bit of complexity in that class, so I would be happy for a second opinion.

My opinion is that we should leave as is (and not introduce enable/disable feature), even if the users cannot run the test from IDE they should be able to generate one. But I'm open to different views on the matter :)